.gitignore

```diff
@@ -54,3 +54,5 @@ cleanup.sh
 docassemble_webapp/docassemble/webapp/static/index.html
 docassemble_webapp/docassemble/webapp/static/css
 docassemble_webapp/docassemble/webapp/static/js
+.history/
+settings.json
```

Comment:
It would be very helpful to add these to the .gitignore--save me from a lot of errors!